### PR TITLE
Fix small typos/capitalization errors 

### PR DESCRIPTION
**Pull request by meghan06**

### CODE REVIEW

File: src/docs/firmware/index.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ next: supported-devices
 # Flashing Custom Firmware
 
 ::: danger
-You are about to make **unofficial** changes to your device! This will likely **void any warranty.** We provide **zero warranty** if you encounter bugs, brick your device, lose your job because the alarm clock didn't go off, or if a thermonuclear war starts. **You are doing all changes at your own risk**!
+You are about to make **unofficial** changes to your device! This will **void any warranty** that came with your original purchae. We provide **zero warranty** if you encounter bugs, brick your device, lose your job because the alarm clock didn't go off, or if a thermonuclear war starts. **You are doing all changes at your own risk**!
```
Comment (ninelore) [inline src/docs/firmware/index.md:8]: Typo

File: src/docs/firmware/battery.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ This method requires opening your device. Be sure to use the correct tools to pr
 2. Take out all of the screws.
 
 ::: tip
-Screws may be underneath "feet", or the material that keeps the Chromebook from sliding.
+Screws may be underneath "feet", or the material (usually rubber) that keeps the Chromebook from sliding.
```
Comment (ninelore) [inline src/docs/firmware/battery.md:17]: Wording on this one is weird in general

**Review by ninelore — CHANGES_REQUESTED** [review]

File: src/docs/firmware/battery.md (hunk 2 above)
Comment (meghan06) [reply src/docs/firmware/battery.md:17]: What about 
"Screws may be underneath "feet", (usually made out of rubber) that keeps the Chromebook from sliding."

Comment (ninelore) [reply src/docs/firmware/battery.md:17]: Dont quote feet, that sounds even more weird. 

"Screws may be underneath the rubber feet that keeps the Chromebook from sliding."

never seen any not made of rubber that worked

**Review by mine-man3000 — APPROVED** [review]
LGTM

**Pull request merged**